README.md

```diff
@@ -145,6 +145,8 @@ Note that,
     * Neon
     * Neon64
 
+Build metrics for Velox are published at <https://facebookincubator.github.io/velox/bm-report/>
```

Comment:
nit: May be say `Build metrics for Velox [can be found here](https://facebookincubator.github.io/velox/bm-report/)`.

Reply:
Was intentionally showing the link url for better visibility.